src/components/settings/globalUpdate/GlobalUpdateSettings.tsx

```diff
@@ -47,7 +48,10 @@ export const GlobalUpdateSettings = () => {
         >
             <GlobalUpdateSettingsInterval />
             <GlobalUpdateSettingsEntries />
-            <GlobalUpdateSettingsCategories />
+            <CategoriesInclusionSetting
+              includeField='includeInUpdate'
+              dialogText={t('library.settings.global_update.categories.label.info')}
```

Comment:
we should add a similar text for the download case

Reply:
Updated. I also moved the other category text used in the CategoriesInclusionSetting component into a new top level `categories` section in the translation JSON files.